types/testingutils/beacon_node.go

```diff
@@ -419,6 +419,9 @@ var TestingProposerDutyFirstSlot = types.ValidatorDuty{
 }
 
 func getValPubKeyByValIdx(valIdx int) phase0.BLSPubKey {
+	if valIdx < 0 {
```

Comment:
Can we just use uint for valIdx?

Reply:
`phase0.ValidatorIndex` is uint64 so I believe I will need to do the conversion either way. But I think your question extends easily to why not use uint64 here as well, right?

Since the goal of this PR was just to solve GoSec issues, I limited myself to this scope. But I agree that in a later PR, we should refactor and use the correct typing where possible, including in these test functions. But this would require much more work now (and I guess there's some priority to fix this and not incur more changes for the `ssv` repo).